amr_kitchen/plotfile_cooker.py

```diff
@@ -319,3 +319,40 @@ def writehdrnewboxes(self, pfdir, boxes, fields):
                         hfile.write(f"{box[i][0]} {box[i][1]}\n")
                 # Write the Level path info
                 hfile.write(f"Level_{lv}/Cell\n")
+
+    def min_max_header(self):
+        """
+        Returns the mins and maxs values for each level and field
+        """
+        all_maxs = []
+        all_mins = []
+        for lv in range(self.limit_level + 1):
+            print('Min Maxes at Level:', lv)
+            all_maxs.append({})
+            all_mins.append({})
+            with open(os.path.join(self.pfile, "Level_"+str(lv), 'Cell_H')) as cpck:
```

Comment:
Replace with:
`open(os.path.join(self.pfile, self.cell_paths[i], "Cell_H"))`
to not hard code the "Level_0, 1, ..." directory structure as it is specified by the global header